api/v1/storagecluster_types.go

```diff
@@ -185,11 +185,13 @@ type ManageCephDashboard struct {
 	SSL bool `json:"ssl,omitempty"`
 }
 
-// ManageCephBlockPools defines how to reconcilea CephBlockPools
+// ManageCephBlockPools defines how to reconcile CephBlockPools
 type ManageCephBlockPools struct {
 	ReconcileStrategy    string `json:"reconcileStrategy,omitempty"`
 	DisableStorageClass  bool   `json:"disableStorageClass,omitempty"`
 	DisableSnapshotClass bool   `json:"disableSnapshotClass,omitempty"`
+	// if set to true, the storageClass created for cephBlockPools will be annotated as the default for the whole cluster
+	DefaultStorageClassForCluster bool `json:"defaultStorageClassForCluster,omitempty"`
```

Comment:
ForCluster is the name that can cause confusion as to whether its defaults for the whole OCP cluster or for the storagecluster, my suggestion is to just have `DefaultStorageClass` as the field name